client/styles/sc-text-paragraph-num-styles.html

```diff
@@ -467,14 +479,30 @@
           background-color: transparent;
         }
 
-        .textual-info-paragraph-inline {
+        .textual-info-paragraph-inline, .textual-info-paragraph-inline.segmented-textual-info-paragraph {
           display: none;
         }
 
         .infomode .textual-info-paragraph-inline {
           display: none;
         }
 
+        .infomode .segmented-textual-info-paragraph {
```

Comment:
can we use list of classes for display: none;
class1,
class2 {}
?

Reply:
I considered it, but I think this has better readability.